app/assets/stylesheets/layout/_layout.scss

```diff
@@ -48,6 +48,26 @@ body {
 
 }
 
+.filters {
+  display: flex;
```

Comment:
Properties should be ordered align-items, display, justify-content, padding-bottom